src/System.Management.Automation/engine/DataStoreAdapterProvider.cs

```diff
@@ -654,5 +663,35 @@ internal PSNoteProperty GetNotePropertyForProviderCmdlets(string name)
             return _noteProperty;
         }
     }
+
+    internal static class ProviderInfoPathSeparators 
+    {
+        public static readonly ReadOnlyCollection<string> FileSystem = new ReadOnlyCollection<string>(
+            new List<string> 
+            { 
+                StringLiterals.DefaultPathSeparatorString, 
+                StringLiterals.AlternatePathSeparatorString
+            });
+
+        public static readonly ReadOnlyCollection<string> Variable = new ReadOnlyCollection<string>(
+            new List<string> 
+            { 
+                StringLiterals.DefaultPathSeparatorString, 
+                StringLiterals.AlternatePathSeparatorString
+            });
+
+        public static readonly ReadOnlyCollection<string> Function = new ReadOnlyCollection<string>(
+            new List<string>
+            {
+                StringLiterals.DefaultPathSeparatorString,
+                StringLiterals.AlternatePathSeparatorString
+            });
```

Comment:
These 3 static members are in fact duplicates. We should have a single static member instead.
I suggest to add the static separator members to [`Separators`](https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/Utils.cs#L1237), for example:
```
internal static readonly ReadOnlyCollection<string> BackAndForwardSlashSeparators = new ReadOnlyCollection...
internal static readonly ReadOnlyCollection<string> BackSlashOnlySeparators =new ReadOnlyCollection...
```
The same `BackAndForwardSlashSeparators` can be used by `FileSystem`, `Variable`, `Function`, `Env`, `Alias`, `Cert`, and `WSMan`.

Reply:
Done